tests/lib/rules/no-misleading-character-class.js

```diff
@@ -85,13 +86,18 @@ ruleTester.run("no-misleading-character-class", rule, {
         {
             code: "var r = /[👍]/",
             errors: [{
+                column: 11,
+                endColumn: 13,
+                line: 1,
                 messageId: "surrogatePairWithoutUFlag",
                 suggestions: [{ messageId: "suggestUnicodeFlag", output: "var r = /[👍]/u" }]
             }]
         },
         {
             code: "var r = /[\\uD83D\\uDC4D]/",
             errors: [{
+                column: 17,
+                endColumn: 23,
```

Comment:
I think the whole sequence `\uD83D\uDC4D` (`column: 11, endColumn: 23`) should be underlined here, as that's the surrogate pair.